cmd/collector/app/grpcserver/grpc_server_test.go

```diff
@@ -38,19 +38,19 @@ func TestFailToListen(t *testing.T) {
 	l, _ := zap.NewDevelopment()
 	handler := app.NewGRPCHandler(l, &mockSpanProcessor{})
 	server := grpc.NewServer()
-	addr, err := StartGRPCCollector(1, server, handler, &mockSamplingStore{}, l, func(e error) {
+	addr, err := StartGRPCCollector(-1, server, handler, &mockSamplingStore{}, l, func(e error) {
```

Comment:
can you elaborate? hard to understand without context

Reply:
also, please define a var `const port = -1` instead of using a constant (but I still don't understand what -1 means)

Reply:
You mean elaborate the reason for using `-1` instead of `1` in a comment in the code? Or just here? I explained a bit in the issue and PR text.